app/code/Tasks/FeedbackManager/Controller/Adminhtml/Manage/MassStatus.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Tasks\FeedbackManager\Controller\Adminhtml\Manage;
+
+use Magento\Backend\App\Action;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Ui\Component\MassAction\Filter;
+use Tasks\FeedbackManager\Model\ResourceModel\Feedback\CollectionFactory;
+
+class MassStatus extends Action
+{
+    protected $collectionFactory;
+
+    protected $filter;
+
+    public function __construct(
+        Context $context,
+        Filter $filter,
+        CollectionFactory $collectionFactory
+    ) {
+        $this->filter = $filter;
+        $this->collectionFactory = $collectionFactory;
+        parent::__construct($context);
+    }
+
+    public function execute()
+    {
+        try {
+            $collection = $this->filter->getCollection($this->collectionFactory->create());
+            $status = $this->getRequest()->getParam('status');
+            if($status == 1) $statusLabel = "approved";
```

Comment:
@gayathri-aravind Pls use match fn instead this way here
https://www.php.net/manual/en/control-structures.match.php

Reply:
Match fn used to get the status label. Commit: 3a2a64414336d13c9a2efa6d09126bb212d438a9